projects/fusion-ui/components/modal/common/base/modal.base.component.ts

```diff
@@ -113,39 +92,27 @@ export class ModalComponent extends FusionBase implements OnInit, OnDestroy, Aft
         }
     }
 
-    /**
-     * remove self from modal service when directive is destroyed
-     */
     ngOnDestroy() {
         this.modalService.remove(this.id);
     }
 
-    /**
-     * open modal
-     */
     open() {
         this.renderer.setStyle(this.elRef.nativeElement, 'display', 'block');
         this.onOpen.emit();
     }
 
-    /**
-     * close modal
-     */
     close(emitEvent = true) {
         this.renderer.setStyle(this.elRef.nativeElement, 'display', 'none');
         if (emitEvent) {
             this.onClose.emit();
         }
     }
 
-    /**
-     * Event on save button
-     */
     save(value) {
         this.onSave.emit(value);
     }
 
-    private setDefaultCssUnit(value: string): string {
+    private static setDefaultCssUnit(value: string): string {
```

Comment:
this shouldn't be a static function and name should start with `get` instead of `set`